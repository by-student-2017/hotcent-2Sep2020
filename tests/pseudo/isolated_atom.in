&CONTROL 
  calculation  = 'scf',
  prefix  = 'Zr'
  outdir  = './work/Zr/'
  pseudo_dir = './' , 
/
&SYSTEM 
  ibrav=1,
  celldm(1)=20.0,
  nat=1,
  ntyp=1,
  nosym=.true.,
  nspin=2,
  starting_magnetization(1) = 1 ,
  tot_magnetization = -1 ,
  ecutwfc=40.0,
  ecutrho=200.0,
  occupations = 'smearing' , 
  smearing = 'gaussian', 
  degauss  = 0.001 , 
/
&ELECTRONS 
  mixing_beta=0.1,
  conv_thr=0.5E-5,
/
ATOMIC_SPECIES 
Zr 91.22400 zr_pbe_v1.uspp.F.UPF
ATOMIC_POSITIONS alat
Zr 0.0 0.0 0.0
K_POINTS automatic
1 1 1  0 0 0
