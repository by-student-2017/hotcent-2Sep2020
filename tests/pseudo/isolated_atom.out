
     Program PWSCF v.6.4.1 starts on 11Sep2020 at 15:52:55 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        6369    5097   1281               382323   273425   34265



     bravais-lattice index     =            1
     lattice parameter (alat)  =      20.0000  a.u.
     unit-cell volume          =    8000.0000 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        12.00
     number of Kohn-Sham states=           10
     kinetic-energy cutoff     =      40.0000  Ry
     charge density cutoff     =     200.0000  Ry
     convergence threshold     =      5.0E-06
     mixing beta               =       0.1000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = SLA  PW   PBE  PBE ( 1  4  3  4 0 0)

     celldm(1)=  20.000000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for Zr read from file:
     ./zr_pbe_v1.uspp.F.UPF
     MD5 check sum: 5db81b1e868ab7776c4564c113de050b
     Pseudo is Ultrasoft + core correction, Zval = 12.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  887 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with  6 coefficients,  rinner =    1.200   1.200   1.200
                                                       1.200   1.200

     atomic species   valence    mass     pseudopotential
        Zr            12.00    91.22400     Zr( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Zr           1.000

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Zr  tau(   1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=     1  gaussian smearing, width (Ry)=  0.0010
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

     Dense  grid:   382323 G-vectors     FFT dimensions: (  96,  96,  96)

     Smooth grid:   273425 G-vectors     FFT dimensions: (  81,  81,  81)

     Estimated max dynamical RAM per process >       1.46 GB
     Generating pointlists ...
     new r_m :   0.4125 (alat units)  8.2500 (a.u.) for type    1

     Initial potential from superposition of free atoms

     starting charge   11.99785, renormalised to   12.00000

     negative rho (up, down):  6.432E-02 6.673E-01
     Starting wfcs are   13 randomized atomic wfcs

     total cpu time spent up to now is        5.7 secs

     Self-consistent Calculation

     iteration #  1     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.5

     negative rho (up, down):  6.372E-02 6.129E-01

     total cpu time spent up to now is       14.8 secs

     total energy              =     -97.48536993 Ry
     Harris-Foulkes estimate   =     -96.95230573 Ry
     estimated scf accuracy    <       1.15823001 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration #  2     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  9.65E-03,  avg # of iterations =  1.0

     negative rho (up, down):  6.276E-02 4.972E-01

     total cpu time spent up to now is       23.1 secs

     total energy              =     -97.94153948 Ry
     Harris-Foulkes estimate   =     -97.50642874 Ry
     estimated scf accuracy    <       0.68553345 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration #  3     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.71E-03,  avg # of iterations =  1.0

     negative rho (up, down):  6.042E-02 5.415E-01

     total cpu time spent up to now is       31.5 secs

     total energy              =     -98.51454359 Ry
     Harris-Foulkes estimate   =     -97.95113939 Ry
     estimated scf accuracy    <       0.43110818 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration #  4     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  3.59E-03,  avg # of iterations =  1.5

     negative rho (up, down):  5.869E-02 5.677E-01

     total cpu time spent up to now is       40.0 secs

     total energy              =     -98.70774027 Ry
     Harris-Foulkes estimate   =     -98.55724024 Ry
     estimated scf accuracy    <       0.17095702 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration #  5     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.42E-03,  avg # of iterations =  2.0

     negative rho (up, down):  5.856E-02 3.401E-01

     total cpu time spent up to now is       48.7 secs

     total energy              =     -98.74841111 Ry
     Harris-Foulkes estimate   =     -98.76330803 Ry
     estimated scf accuracy    <       0.42803671 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.10 Bohr mag/cell

     iteration #  6     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.42E-03,  avg # of iterations =  1.0

     negative rho (up, down):  5.913E-02 2.032E-01

     total cpu time spent up to now is       57.6 secs

     total energy              =     -98.65269872 Ry
     Harris-Foulkes estimate   =     -98.74997834 Ry
     estimated scf accuracy    <       0.37906966 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.06 Bohr mag/cell

     iteration #  7     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.42E-03,  avg # of iterations =  3.0

     negative rho (up, down):  6.055E-02 2.041E-01

     total cpu time spent up to now is       66.6 secs

     total energy              =     -98.66564824 Ry
     Harris-Foulkes estimate   =     -98.68432632 Ry
     estimated scf accuracy    <       0.06345855 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.02 Bohr mag/cell

     iteration #  8     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.29E-04,  avg # of iterations =  3.0

     negative rho (up, down):  6.056E-02 1.746E-01

     total cpu time spent up to now is       75.7 secs

     total energy              =     -98.69949039 Ry
     Harris-Foulkes estimate   =     -98.69848454 Ry
     estimated scf accuracy    <       0.10004327 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration #  9     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.29E-04,  avg # of iterations =  1.0

     negative rho (up, down):  6.042E-02 1.188E-01

     total cpu time spent up to now is       84.7 secs

     total energy              =     -98.69884649 Ry
     Harris-Foulkes estimate   =     -98.69958086 Ry
     estimated scf accuracy    <       0.09816278 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 10     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.29E-04,  avg # of iterations =  1.0

     negative rho (up, down):  5.973E-02 1.176E-01

     total cpu time spent up to now is       93.7 secs

     total energy              =     -98.67765912 Ry
     Harris-Foulkes estimate   =     -98.69942717 Ry
     estimated scf accuracy    <       0.07924834 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 11     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.29E-04,  avg # of iterations =  1.5

     negative rho (up, down):  5.981E-02 1.064E-01

     total cpu time spent up to now is      102.7 secs

     total energy              =     -98.68217233 Ry
     Harris-Foulkes estimate   =     -98.68301101 Ry
     estimated scf accuracy    <       0.02187731 Ry

     total magnetization       =     3.97 Bohr mag/cell
     absolute magnetization    =     3.97 Bohr mag/cell

     iteration # 12     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.82E-04,  avg # of iterations =  1.0

     negative rho (up, down):  6.053E-02 9.103E-02

     total cpu time spent up to now is      111.7 secs

     total energy              =     -98.68127394 Ry
     Harris-Foulkes estimate   =     -98.68237921 Ry
     estimated scf accuracy    <       0.01652856 Ry

     total magnetization       =     3.81 Bohr mag/cell
     absolute magnetization    =     3.81 Bohr mag/cell

     iteration # 13     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.38E-04,  avg # of iterations =  8.5

     negative rho (up, down):  5.954E-02 9.262E-02

     total cpu time spent up to now is      122.0 secs

     total energy              =     -98.67489279 Ry
     Harris-Foulkes estimate   =     -98.68496557 Ry
     estimated scf accuracy    <       0.04433574 Ry

     total magnetization       =     3.99 Bohr mag/cell
     absolute magnetization    =     3.99 Bohr mag/cell

     iteration # 14     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.38E-04,  avg # of iterations =  2.0

     negative rho (up, down):  5.944E-02 9.013E-02

     total cpu time spent up to now is      131.2 secs

     total energy              =     -98.68350417 Ry
     Harris-Foulkes estimate   =     -98.68297604 Ry
     estimated scf accuracy    <       0.00481423 Ry

     total magnetization       =     3.45 Bohr mag/cell
     absolute magnetization    =     3.45 Bohr mag/cell

     iteration # 15     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  2.0

     negative rho (up, down):  5.925E-02 1.016E-01

     total cpu time spent up to now is      140.2 secs

     total energy              =     -98.68608271 Ry
     Harris-Foulkes estimate   =     -98.68366899 Ry
     estimated scf accuracy    <       0.00568194 Ry

     total magnetization       =     3.35 Bohr mag/cell
     absolute magnetization    =     3.35 Bohr mag/cell

     iteration # 16     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  5.937E-02 1.098E-01

     total cpu time spent up to now is      149.3 secs

     total energy              =     -98.69142806 Ry
     Harris-Foulkes estimate   =     -98.68867210 Ry
     estimated scf accuracy    <       0.00900344 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 17     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  5.996E-02 1.037E-01

     total cpu time spent up to now is      158.5 secs

     total energy              =     -98.69229718 Ry
     Harris-Foulkes estimate   =     -98.69162256 Ry
     estimated scf accuracy    <       0.00584563 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 18     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  6.041E-02 9.308E-02

     total cpu time spent up to now is      167.5 secs

     total energy              =     -98.69152802 Ry
     Harris-Foulkes estimate   =     -98.69235625 Ry
     estimated scf accuracy    <       0.00624631 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 19     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  6.036E-02 9.368E-02

     total cpu time spent up to now is      176.4 secs

     total energy              =     -98.69299807 Ry
     Harris-Foulkes estimate   =     -98.69167840 Ry
     estimated scf accuracy    <       0.00954338 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 20     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  6.040E-02 9.049E-02

     total cpu time spent up to now is      185.4 secs

     total energy              =     -98.69324017 Ry
     Harris-Foulkes estimate   =     -98.69305487 Ry
     estimated scf accuracy    <       0.00795104 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 21     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  6.043E-02 9.488E-02

     total cpu time spent up to now is      194.3 secs

     total energy              =     -98.69274251 Ry
     Harris-Foulkes estimate   =     -98.69324361 Ry
     estimated scf accuracy    <       0.00791546 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 22     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  1.0

     negative rho (up, down):  5.985E-02 9.242E-02

     total cpu time spent up to now is      203.2 secs

     total energy              =     -98.69545782 Ry
     Harris-Foulkes estimate   =     -98.69275902 Ry
     estimated scf accuracy    <       0.00785075 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 23     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.01E-05,  avg # of iterations =  2.0

     negative rho (up, down):  5.956E-02 2.259E-01

     total cpu time spent up to now is      212.3 secs

     total energy              =     -98.71902237 Ry
     Harris-Foulkes estimate   =     -98.69793628 Ry
     estimated scf accuracy    <       0.00061685 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 24     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.14E-06,  avg # of iterations =  4.5

     negative rho (up, down):  5.960E-02 1.566E-01

     total cpu time spent up to now is      222.1 secs

     total energy              =     -98.71405863 Ry
     Harris-Foulkes estimate   =     -98.71986668 Ry
     estimated scf accuracy    <       0.00089606 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 25     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.14E-06,  avg # of iterations =  3.0

     negative rho (up, down):  5.965E-02 1.118E-01

     total cpu time spent up to now is      231.3 secs

     total energy              =     -98.70784720 Ry
     Harris-Foulkes estimate   =     -98.71413762 Ry
     estimated scf accuracy    <       0.00042294 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 26     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  3.52E-06,  avg # of iterations =  3.0

     negative rho (up, down):  5.989E-02 1.538E-01

     total cpu time spent up to now is      240.6 secs

     total energy              =     -98.71124949 Ry
     Harris-Foulkes estimate   =     -98.70789595 Ry
     estimated scf accuracy    <       0.00027375 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 27     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  2.28E-06,  avg # of iterations =  3.0

     negative rho (up, down):  6.021E-02 1.614E-01

     total cpu time spent up to now is      249.9 secs

     total energy              =     -98.69785100 Ry
     Harris-Foulkes estimate   =     -98.71137841 Ry
     estimated scf accuracy    <       0.00066941 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 28     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  2.28E-06,  avg # of iterations =  4.0

     negative rho (up, down):  6.011E-02 6.349E-02

     total cpu time spent up to now is      259.5 secs

     total energy              =     -98.69793704 Ry
     Harris-Foulkes estimate   =     -98.69808817 Ry
     estimated scf accuracy    <       0.00018998 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 29     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.58E-06,  avg # of iterations =  2.5

     negative rho (up, down):  6.011E-02 6.294E-02

     total cpu time spent up to now is      268.7 secs

     total energy              =     -98.69797928 Ry
     Harris-Foulkes estimate   =     -98.69796739 Ry
     estimated scf accuracy    <       0.00001112 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 30     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  9.27E-08,  avg # of iterations =  2.0

     negative rho (up, down):  6.013E-02 1.235E-01

     total cpu time spent up to now is      277.7 secs

     total energy              =     -98.69804113 Ry
     Harris-Foulkes estimate   =     -98.69798075 Ry
     estimated scf accuracy    <       0.00000955 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     iteration # 31     ecut=    40.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  7.96E-08,  avg # of iterations =  2.5

     negative rho (up, down):  6.018E-02 3.870E-01

     Magnetic moment per site:
     atom:    1    charge:   11.9899    magn:    3.9899    constr:    0.0000

     total cpu time spent up to now is      286.2 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 34265 PWs)   bands (ev):

   -52.1393 -30.3332 -30.3295 -30.3275  -3.8338  -2.6516  -2.6496  -2.6406
    -1.9509  -1.9476

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 34265 PWs)   bands (ev):

   -50.8920 -28.8647 -28.8581 -28.8471  -1.7391  -0.6299  -0.6283  -0.2079
    -0.2000  -0.1827

     the Fermi energy is    -2.1872 ev

!    total energy              =     -98.70091693 Ry
     Harris-Foulkes estimate   =     -98.69804195 Ry
     estimated scf accuracy    <       0.00000418 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -115.85057898 Ry
     hartree contribution      =      55.24203915 Ry
     xc contribution           =     -17.66383519 Ry
     ewald contribution        =     -20.42854192 Ry
     smearing contrib. (-TS)   =      -0.00000000 Ry

     total magnetization       =     4.00 Bohr mag/cell
     absolute magnetization    =     4.00 Bohr mag/cell

     convergence has been achieved in  31 iterations

     Writing output data file Zr.save/

     init_run     :      3.91s CPU      5.61s WALL (       1 calls)
     electrons    :    203.02s CPU    280.59s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.64s CPU      0.68s WALL (       1 calls)
     potinit      :      1.38s CPU      1.63s WALL (       1 calls)
     hinit0       :      0.72s CPU      0.81s WALL (       1 calls)

     Called by electrons:
     c_bands      :     35.56s CPU     36.04s WALL (      31 calls)
     sum_band     :     70.14s CPU    107.64s WALL (      31 calls)
     v_of_rho     :     53.31s CPU     56.37s WALL (      32 calls)
     newd         :     34.53s CPU     72.29s WALL (      32 calls)
     mix_rho      :      8.59s CPU      8.74s WALL (      31 calls)

     Called by c_bands:
     init_us_2    :      1.45s CPU      1.48s WALL (     126 calls)
     cegterg      :     33.38s CPU     33.84s WALL (      62 calls)

     Called by sum_band:
     sum_band:bec :      0.00s CPU      0.00s WALL (      62 calls)
     addusdens    :     59.47s CPU     96.93s WALL (      31 calls)

     Called by *egterg:
     h_psi        :     30.86s CPU     31.08s WALL (     197 calls)
     s_psi        :      0.53s CPU      0.48s WALL (     197 calls)
     g_psi        :      0.12s CPU      0.14s WALL (     133 calls)
     cdiaghg      :      0.06s CPU      0.04s WALL (     195 calls)

     Called by h_psi:
     h_psi:pot    :     30.77s CPU     30.86s WALL (     197 calls)
     h_psi:calbec :      0.66s CPU      0.61s WALL (     197 calls)
     vloc_psi     :     29.59s CPU     29.77s WALL (     197 calls)
     add_vuspsi   :      0.52s CPU      0.49s WALL (     197 calls)

     General routines
     calbec       :      0.86s CPU      0.83s WALL (     259 calls)
     fft          :     20.62s CPU     20.90s WALL (     794 calls)
     ffts         :      1.47s CPU      1.56s WALL (     126 calls)
     fftw         :     32.09s CPU     32.18s WALL (    3710 calls)
     interpolate  :      2.58s CPU      2.54s WALL (      64 calls)

     Parallel routines

     PWSCF        :   3m26.98s CPU   4m46.34s WALL


   This run was terminated on:  15:57:41  11Sep2020            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
